config/services.php

```diff
@@ -13,8 +13,6 @@
         ->set('env(TPAY_CLIENT_SECRET)', '')
         ->set('env(TPAY_CARDS_API)', '')
         ->set('env(TPAY_NOTIFICATION_SECURITY_CODE)', '')
-        ->set('env(TPAY_MERCHANT_ID)', '')
-        ->set('env(TPAY_GOOGLE_MERCHANT_ID)', '')
```

Comment:
Can we remove all parameters from here as they are defined in the `config/config.php`?

Reply:
Rebase left-over 😅